jml/run-proof.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e -u -o pipefail -x
+trap "kill 0" SIGINT SIGTERM
+
+mkdir -p /var/tmp/openjml{,-build}
+
+# Install OpenJML
+pushd /var/tmp/openjml-build
+git clone https://github.com/OpenJML/OpenJML.git
+cd OpenJML && git checkout e4dc6371e2cf45d1a08b5c91990e2a875f33dd27
```

Comment:
This seems really brittle. What is this commit supposed to be?

Reply:
I believe this makes the build less brittle.  When OpenJML changes, we don't want the proof to fail because it can no longer check this proof.  I can check out the tip of the branch which is supporting these builds, but our build can then break without any changes on our side.  (This is similar to builds that blindly pull from apt/yum repos.)  I'm happy to change it to pull a branch if you like, with that caveat.

Reply:
Can we use a tag instead? This would let us do a shallow clone to reduce load on github's servers, and also make it more obvious which version we're referencing.